xpholder/commands/mod/EditOtherCharacter.js

```diff
@@ -0,0 +1,134 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { sqlInjectionCheck, buildCharacterEmbed } = require("../../utils");
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('edit__other_character')
+        .setDescription('Edit Another Player\'s Character [MOD]')
+        .addUserOption(option => option
+            .setName("player")
+            .setDescription("The Player You Wish To Edit")
+            .setRequired(true))
+        .addIntegerOption(option => option
+            .setName("character")
+            .setDescription("Which Character You Want To Edit ( 1 -> 10 )")
+            .setMinValue(1)
+            .setMaxValue(10)
+            .setAutocomplete(true)
+            .setRequired(true))
+        .addStringOption(option => option
+            .setName("character_name")
+            .setDescription("Name Of The Character")
+            .setRequired(true))
+
+        .addStringOption(option => option
+            .setName("sheet_url")
+            .setDescription("A Link To Their Character Sheet")
+            .setRequired(false))
+        .addStringOption(option => option
+            .setName("picture_url")
+            .setDescription("A Link To The Character Picture")
+            .setRequired(false))
+
+        .addBooleanOption(option => option
+            .setName("public")
+            .setDescription("Show This Command To Everyone?")
+            .setRequired(false))
+    ,
+    async execute(guildService, interaction) {
+        /*
+        ----------
+        VALIDATION
+        ----------
+        */
+
+        if (!guildService.isMod(interaction.member._roles) 
+            && interaction.user.id != interaction.guild.ownerId
+            && !guildService.isDev(interaction.member._roles)) {
+            await interaction.editReply("Sorry, you do not have the right role to use this command.");
+            return;
+        }
+
+        /*
+        --------------
+        INITALIZATIONS
+        --------------
+        */
+        const player = interaction.options.getUser("player");
+        const characterNumber = interaction.options.getInteger("character");
+        const name = interaction.options.getString("character_name");
+        const sheetUrl = interaction.options.getString("sheet_url");
+        const pictureUrl = interaction.options.getString("picture_url");
+
+        const guild = interaction.member.guild
```

Comment:
player needs to be defined after guild because you are only getting a partial player object here. to get the full object you need something like this:

```
const player = await guild.members.fetch(
      interaction.options.getUser("player").id
    );
```

Reply:
Done.